templates/src/objective-c/BoringSSL-GRPC.podspec.template-e

```diff
@@ -0,0 +1,249 @@
+%YAML 1.2
```

Comment:
what is this file with `.template-e` suffix?

Reply:
I'm not sure... probably created a copy with a typo. I'm removing this file.